core/src/main/resources/reference.conf

```diff
@@ -144,3 +144,12 @@ sam.timeout = 3 minutes
 prometheus {
   endpointPort = 9098
 }
+
+multiCloudWorkspaces {
+    azureConfig {
+        costSavingLandingZoneParameters = {
+          "AKS_COST_SAVING_SPOT_NODES_ENABLED": "true"
```

Comment:
My understanding was that the cost savings VPA + spot nodes stuff was going to be opt-in, is that the case? If so these should parameters should default to false.

Reply:
yes thank you, changed. i had modified it in a couple places and forgot to change this back.

Reply:
Oh jeez vacation brain, nvm i actually have to reverse this. 
These actually are true by default. The check that enables and unenables them is actually in BpmBillingProjectLifecycle. It checks the user set flag and either pulls these true parameters, or doesn't even use them if the cost savings flag is false.